src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Lottery extends RandoM {
+    public String getRandomBall() {
+        return "color of the ball: " + getRandomColor() + ", number of the ball: "
+                + getRandomNumber() + ".";
```

Comment:
According to the checklist, don't create temporary variables if you can directly return value from the method. Instead of concatenating strings, you can directly return the result. Also, consider renaming the `getRandomBall` method to reflect that it returns a string representation of the ball, not an actual Ball object.